### PR TITLE
fix: change cards behavior on hover, replace Angular Grid-List with flexbox

### PR DESCRIPTION
**Pull request by NSakuta**
# Issue
https://amiconsult.atlassian.net/browse/BTB-XX

## Changes

- 
- 

## Checklist

* [x] I tested the functionality on my local environment.
* [x] I made sure my code is easy to read and understand.
* [x] I provided sufficient automated tests for the code.
* [x] There a no conflicts from my-branch..main.

### CODE REVIEW

File: src/app/search-page/search-result.component/search-result.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-  <mat-grid-tile *ngFor="let product of visibleProducts" (click)='toProductDetailPage(product.id)' data-id="product.id">
+<div class="wrapper">
+  <div class="card-wrapper" *ngFor="let product of visibleProducts" (click)='toProductDetailPage(product.id)' data-id="product.id">
 
     <app-card [product]="product"></app-card>
 
-  </mat-grid-tile>
-</mat-grid-list>
+  </div>
+</div>
```
Comment (knicken) [inline src/app/search-page/search-result.component/search-result.component.html:7]: Why did you remove the grid-list and wrote it yourself?

File: src/shared/card-component/card.component.scss [hunk 2]
```diff
@@ -5,10 +5,13 @@
   margin: 0 auto;
   height: 90%;
   width: 90%;
-  transition: 1s ease;
+  box-shadow: rgba(0, 0, 0, 0.06) 0px 2px 4px;
+  transition: 0.5s ease;
 }
 .card:hover {
-  transform: scale(0.97);
+  box-shadow: rgba(0, 0, 0, 0.22) 0px 19px 43px;
```
Comment (knicken) [inline src/shared/card-component/card.component.scss:12]: Where did you get "19px 43px" from? The Material guide says otherwise.
_We don't need pixel density into consideration here -> 8dp = 8px_

**Review by knicken — CHANGES_REQUESTED** [review]

File: src/shared/card-component/card.component.scss (hunk 2 above)
Comment (NSakuta) [reply src/shared/card-component/card.component.scss]: changed it

**Review by knicken — APPROVED** [review]
